airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1184,6 +1184,29 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/source_oauths/oauth_params/create:
```

Comment:
these were just moved for grouping with other oauth endpoints. no actual changes happened here.